dbms/src/Flash/Coprocessor/TablesRegionsInfo.cpp

```diff
@@ -47,6 +48,8 @@ const SingleTableRegions & TablesRegionsInfo::getTableRegionInfoByTableID(Int64
 static bool needRemoteRead(const RegionInfo & region_info, const TMTContext & tmt_context)
 {
     fiu_do_on(FailPoints::force_no_local_region_for_mpp_task, { return true; });
+    if (tmt_context.getContext().isDisaggregatedComputeMode())
+        return true;
```

Comment:
Why `true` for compute mode?

Reply:
For tiflash_compute node, we record all region info into `table_region_info.remote_regions` (by checking `needRemoteRead()` is true or not), then MPPTask will be constructed and sent to tiflash_storage node which store `remote_regions`.